src/main/java/Backend.java

```diff
@@ -0,0 +1,64 @@
+import java.io.IOException;
+import java.io.File;
+import java.util.Scanner;
```

Comment:
Java Coding Standard: The ordering of import statements must be consistent.

In this case you could consider grouping your `io` and `util` together.